rholang/src/main/scala/coop/rchain/rholang/interpreter/Reduce.scala

```diff
@@ -182,8 +182,13 @@ class DebruijnInterpreter[M[_], F[_]](
       else if (terms.size > 256) rand.splitShort(id.toShort)
       else rand.splitByte(id.toByte)
 
-    val indexedTerms = terms.zipWithIndex.toList
-    if (terms.size <= Reduce.parallelism) {
+    val indexedTerms   = terms.zipWithIndex.toList
+    val termSplitLimit = Short.MaxValue + 1
```

Comment:
this should say `val termSplitLimit = Short.MaxValue`

Reply:
Should this be the same as for bytes, to detect overflow? Limit is the count and not the index. Am I missing something?